### PR TITLE
Add sphinx-relay app

### PR DESCRIPTION
**Pull request by gonzaloaune**
# App Submission

### App name
Sphinx Relay

### Version
v1.2.9

### One line description of the app
Chat & pay over the bitcoin lightning network.

### Summary of the app
Sphinx relay turns your lightning node into a personal communication server. Messages are end to end encrypted and transmitted over the bitcoin lightning network. You can join tribes at https://tribes.sphinx.chat. If you join a podcast tribe, you can listen to the podcast in Sphinx and stream donations to the host.

### Developed by
Stakwork

### Developer website
https://sphinx.chat

### Source code repository
https://github.com/stakwork/sphinx-relay

### Support link
https://t.me/joinchat/Fb4OcRo4LrRD6NXfsNdeeQ

### Uses
- [ ] Bitcoin Core
- [ ] Electrum server
- [X] LND

### 256x256 SVG icon
https://svgshare.com/i/Sa4.svg

### App screenshots
1_chat_promo
2_tribes_promo
3_podcast_promo

### I have tested my app on:
- [X] [Umbrel dev environment](https://github.com/getumbrel/umbrel-dev)
- [x] [Umbrel OS on a Raspberry Pi 4](https://github.com/getumbrel/umbrel-os)
- [ ] [Custom Umbrel install on Linux](https://github.com/getumbrel/umbrel#-installation)

### CODE REVIEW

**Comment by Antanasijevic** [comment]
Logo 256x256
Link to SVG icon
https://drive.google.com/file/d/1Jt6nH3rjFT4A8nrE5cDn1s0GxvY6VOx7/view?usp=sharing

Icon_256x256

File: scripts/update/.updateignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.*
+/.*
```
Comment (AaronDewes) [inline scripts/update/.updateignore:1]: This would be better in my opinion
```suggestion
.*
!.gitkeep
```

**Comment by pitoi** [comment]
Description:
Sphinx relay turns your lightning node into a personal communication server. Messages are end to end encrypted and transmitted over the bitcoin lightning network. You can join tribes at https://tribes.sphinx.chat. If you join a podcast tribe, you can listen to the podcast in Sphinx and stream donations to the host. 

Support links:
https://t.me/joinchat/Fb4OcRo4LrRD6NXfsNdeeQ
support@stakwork.com

Screenshots:
1_chat_promo
2_tribes_promo
3_podcast_promo

**Review by lukechilds — COMMENTED** [review]
Code wise this all looks great!

I've setup the required constants and pinned the Docker images.

I've tested this on umbrel-dev and my mainnet Umbrel and both are working great.

It pairs fine with the Sphinx app but I get errors when trying to send messages, I'm unsure if that is related to the Umbrel integration or some unrelated problem.

It would be great to get some feedback on how I can resolve this to test everything is fully working.

Other than that all good, great work!

**Review by mayankchhabra — APPROVED** [review]
Great job on packaging this, @gonzaloaune and the entire Sphinx team! The integration is really slick. Just tried it on a new mainnet Umbrel install and the connection was quick and easy. 

Will try out the chat feature as a part of our E2E testing before the final release. Since @lukechilds have already tried that out, we should be good to merge!

**Comment by lukechilds** [comment]
Merged, great work guys!

**Pull request merged**